compiler/rustc_target/src/spec/targets/m68k_unknown_linux_gnu.rs

```diff
@@ -12,11 +12,18 @@ pub(crate) fn target() -> Target {
             description: Some("Motorola 680x0 Linux".into()),
             tier: Some(3),
             host_tools: Some(false),
-            std: None, // ?
+            std: Some(false), // std cannot be linked for now
         },
         pointer_width: 32,
         data_layout: "E-m:e-p:32:16:32-i8:8:8-i16:16:16-i32:16:32-n8:16:32-a:0:16-S16".into(),
         arch: "m68k".into(),
-        options: TargetOptions { endian: Endian::Big, mcount: "_mcount".into(), ..base },
+        options: TargetOptions {
+            endian: Endian::Big,
+            mcount: "_mcount".into(),
+
+            // LLD currently does not have support for M68k
+            linker: Some("m68k-linux-gnu-ld".into()),
```

Comment:
Something like `link_self_contained: LinkSelfContainedDefault::False` instead may work too. This will also ensure that attempting to enable self-contained linking using lld on the cli gives an error.

Reply:
Ah, thank you!

Reply:
@knickish Can you try this?

Reply:
Yup, doing a build right now

Reply:
This worked, `core` and everything besides `std` built and linked; the `std` link failed with the exact same error as with `linker: Some("m68k-linux-gnu-ld".into())`.